python/ray/data/_internal/util.py

```diff
@@ -573,7 +573,10 @@ def get_compute_strategy(
     fn: "UserDefinedFunction",
     fn_constructor_args: Optional[Iterable[Any]] = None,
     compute: Optional[Union[str, "ComputeStrategy"]] = None,
-    concurrency: Optional[Union[int, Tuple[int, int], Tuple[int, int, int]]] = None,
+    concurrency: Optional[int] = None,
+    initial_concurrency: Optional[int] = None,
+    min_concurrency: Optional[int] = None,
+    max_concurrency: Optional[int] = None,
```

Comment:
@owenowenisme sorry for confusion, but we'd need to rename this to be 

 - initial_op_concurrency
 - min_op_concurrency
 - max_op_concurrency

Reason for that is that `max_concurrency` is Ray Core parameter that some APIs (like map_batches) accept as kwarg and hence we'd have to rename it to make sure there's no breakage

Reply:
BTW, i've updated the original ticket w/ this proposal PTAL